docs/reference/commands/cluster.rst

```diff
@@ -0,0 +1,210 @@
+===========
+``cluster``
+===========
+
+Manage the MicroCeph cluster
```

Comment:
Let's use this as a model to be applied everywhere:

**Manages the MicroCeph cluster.**

So there's an 's' at the end of the first word and a '.' at the end of the last word.